### PR TITLE
Update default.html

### PR DESCRIPTION
**Pull request by yuktibagrodia**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #

Ref #

#### Changes done:
- [x] Task 1

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -191,7 +203,7 @@
         <div id="mainhome" class="main-content">
           <div id="content">
             <nav class="navbar navbar-expand-lg navbar-light bg-light container-fluid">
-                <button type="button" id="sidebarCollapse" class="btn btn-info new mr-3">
+                <button class="button" type="button" id="sidebarCollapse" class="btn btn-info new mr-3">
```
Comment (Shivansh2407) [inline _layouts/default.html:206]: There is already a Class Attribute Mentioned in the Button Tag, Please do add this button class in it. Also please add the before and after screenshots to show the difference.

**Review by Shivansh2407 — CHANGES_REQUESTED** [review]

**Comment by Vishal-raj-1** [comment]
@yuktibagrodia Any update ? Do you want to do this. I think you just need to  `<button type="button" id="sidebarCollapse" class="btn btn-info new mr-3 button">`

**Comment by yuktibagrodia** [comment]
@Vishal-raj-1 you can go-ahead

**Comment by Vishal-raj-1** [comment]
> @Vishal-raj-1 you can go-ahead

Thanks for your reply!!

**Pull request closed without merging**